src/axom/sidre/tests/spio/spio_parallel.hpp

```diff
@@ -186,7 +186,7 @@ TEST(spio_parallel, parallel_writeread)
    */
   IOManager reader(MPI_COMM_WORLD);
 
-  reader.read(ds2->getRoot(), root_name);
+  reader.read(ds2->getRoot(), root_name, PROTOCOL);
```

Comment:
This test is currently failing in axom configurations that do not have `hdf5`.

The following issues track this:  
* #1018
* https://github.com/LLNL/conduit/issues/1075

Reply:
Kenny, what is your plan for this?  Is there a way to disable line 195 for build configurations that don't have hdf5?

Reply:
@cyrush pointed out that this has likely already been resolved in `conduit@0.8.4:` and we're on `conduit@0.8.3`, so it might be fixed by just upgrading.

Reply:
More directly, the `sidre_parallel` test is not currently failing in axom@develop or with this branch since we're not testing any configurations without `hdf5`

Reply:
This was resolved by updating to `conduit@0.8.6` in #1021